### ISSUE TITLE
ERROR : python-software-properties package was unable to be installed ubuntu 18.04

### ISSUE THREAD

**Issue by khanfar** [RangeNetworks/dev#93]
help i got error when try  ./build.sh N210
i got error :

mwk@ubuntu:~/dev$ ./build.sh N210
# checking for a supported radio type
# - found
# checking for a compatible build host
# - WARNING : dev-tools is currently only tested on Ubuntu 16.04, YMMV. Please open an issue if you've used it successfully on another version of Ubuntu.
#
# adding additional repo tools
dpkg-query: package 'python-software-properties' is not installed and no information is available
Use dpkg --info (= dpkg-deb --info) to examine archive files,
and dpkg --contents (= dpkg-deb --contents) to list their contents.
# - missing python-software-properties, installing dependency
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package python-software-properties is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  software-properties-common

E: Package 'python-software-properties' has no installation candidate
# - ERROR : python-software-properties package was unable to be installed
mwk@ubuntu:~/dev$

**Comment by sharad1126** [comment]
I think you should just install software-properties-common and upgrade this. Then comment out python-software-properties in build.sh and it should work. But I got an error for `libortp9`. I am using Ubuntu 20.04!

**Comment by ruffm** [comment]
> I think you should just install software-properties-common and upgrade this. Then comment out python-software-properties in build.sh and it should work. But I got an error for `libortp9`. I am using Ubuntu 20.04!

Yeah, I kept going down the "upgrade and comment out" route but at some point, it gets kind of ridiculous.
